packages/nc-gui-v2/app.vue

```diff
@@ -30,20 +30,21 @@ const toggleSidebar = () => {
 
       <div class="flex-1" />
 
-      <div class="ml-4 flex justify-center flex-1">
+      <div class="ml-4 flex justify-center flex-shrink">
```

Comment:
`flex-shrink` is a vuetify style I assume - replace with [TailwindCSS shrink class](https://tailwindcss.com/docs/flex-shrink)